WebApiCustomers/Controllers/CustomersController.cs

```diff
@@ -0,0 +1,78 @@
+﻿
+using Microsoft.AspNetCore.Mvc;
+using System.Xml.Linq;
+using WebApiCustomers.Model;
+using WebApiCustomers.Repository;
+
+namespace WebApiCustomers.Controllers
+{
+    [Route("Api/[controller]")]
+    [ApiController]
+    public class CustomersController : Controller
+    {
+        private readonly IBaseRepository _repository;
+
+        public CustomersController(IBaseRepository repository)
+        {
+            _repository = repository;
+        }
+
+        [HttpGet]
+        public IActionResult Get()
+        {
+            return Ok(_repository.GetAll());
+        }
+
+        [HttpGet("{id}")]
+        public IActionResult GetById(int id)
+        {
+            var response = _repository.GetById(id);
+
+            if (response != null)
+            {
+                return Ok(response);
+            }
+            return NotFound($"Id not found: {id}");
+
+        }
+
+        [HttpPost]
+        public IActionResult Post(CustomersModel model)
+        {
+            var response = _repository.Create(model);
+
+            if (response == 201)
+            {
+                return Created("", response);
+            }
+            return Conflict("There is already a customer with this CPF and Email ");
+
+
+        }
```

Comment:
```suggestion
        public IActionResult Post(CustomersModel model)
        {
            var response = _repository.Create(model);

            if (response == 201)
            {
                return Created("", response);
            }
            return BadRequest("There is already a customer with this CPF and Email ");
        }
```
Cuidar com as linhas em branco desnecessárias e seria melhor retornar BadRequest